src/org/omegat/gui/notes/NotesTextArea.java

```diff
@@ -137,12 +135,12 @@ public void redo() {
     public void populatePaneMenu(JPopupMenu menu) {
         final JMenuItem notify = new JCheckBoxMenuItem(OStrings.getString("GUI_NOTESWINDOW_NOTIFICATIONS"));
         notify.setSelected(Preferences.isPreference(Preferences.NOTIFY_NOTES));
-        notify.addActionListener(new ActionListener() {
-            @Override
-            public void actionPerformed(ActionEvent e) {
-                Preferences.setPreference(Preferences.NOTIFY_NOTES, notify.isSelected());
-            }
-        });
+        notify.addActionListener(e -> Preferences.setPreference(Preferences.NOTIFY_NOTES, notify.isSelected()));
         menu.add(notify);
     }
+    
+    @Override
+    public void requestFocus() {
+    	scrollPane.getViewport().getView().requestFocusInWindow();
```

Comment:
With enhancement #349, we can just add `requestVisible`

```suggestion
        requestVisible();
    	scrollPane.getViewport().getView().requestFocusInWindow();
```

Reply:
```suggestion
        StaticUIUtils.requestVisible(scrollPane);
    	scrollPane.getViewport().getView().requestFocusInWindow();
```

Reply:
@briacp please try the change on your environment?

Reply:
That looks good to me, the `requestVisible` call works nicely.

Reply:
What's the status for this feature?

Reply:
waiting @briacp  to add a feature to restore pane by shortcut when note is invisible.